Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -4864,6 +4865,28 @@ def com_google_fonts_check_cjk_not_enough_glyphs(ttFont):
         yield PASS, "Font has the correct quantity of CJK glyphs"
 
 
+@check(
+    id = 'com.google.fonts/check/cjk_chws_feature',
+    conditions = ['is_cjk_font'],
+    rationale = """
+        The W3C recommends the addition of chws and vchw features to CJK fonts to enhance the spacing of glyphs in environments which do not fully support JLREQ layout rules.
+
+        The chws_tool utility (https://github.com/googlefonts/chws_tool) can be used to add these features automatically.
+    """,
+    proposal = 'https://github.com/googlefonts/fontbakery/issues/3363'
+)
+def com_google_fonts_check_cjk_chws_feature(ttFont):
+    """Does the font contain chws and vchw features?"""
+    from fontbakery.profiles.layout import feature_tags
+    tags = feature_tags(ttFont)
+    if "chws" not in tags or "vchw" not in tags:
+        yield WARN,\
+              Message('cjk-no-chws',
```

Comment:
I'd suggest here `'missing-chws-feature'` and `'missing-vchw-feature'` as a more readable pair of keywords.

Reply:
please address this